package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "soundclout-active-player-module",
+  "version": "1.0.0",
+  "description": "This features an animated player synced proportionally with a time-stamped location in a song. Its clickable components include a play/pause toggle button, fastforward/rewind buttons, shuffle/repeat buttons, hover volume control, album thumbnails, a like button, and a \"up next\" pop up menu.  The \"up next\" pop up menu also features a button to clear the list, a toggle to autoplay upcoming tracks, as well as like buttons and a pop-up options menu for each track.  The menu allows you to like, share, add or remove from \"next up\", add to a playlist, or start a radio station of related tracks.",
+  "main": "index.js",
```

You should specify the file path for this line. Currently I am unaware if this is a reference to your /server/index.js file or /database/index.js file.

```suggestion
  "main": "/server/index.js",
```

---

Cool! Will do

---

I ended up leaving this one because when I read about the [main](https://nodesource.com/blog/the-basics-of-package-json-in-node-js-and-npm/) property, it was unclear to me if this is necessary.  Feel free to take a look at the link and let me know what you think.

---

It's not a big deal unless you are referencing it somewhere else (which I don't believe you are). I'll approve of it, as I don't think it will really matter, but it still is a little ambiguous.